gpt_index/readers/file/tabular_parser

```diff
@@ -0,0 +1,30 @@
+"""Tabular parser.
```

Comment:
can you make this have a .py extension?

Reply:
woops, yep on it

Reply:
Also if I understand correctly, you want a sceenshot of a csv parsed to text?

Reply:
![image](https://user-images.githubusercontent.com/43042800/213862247-a95a3599-81cd-431a-af42-5b0b0f1d225b.png)